### PR TITLE
change moderator name

### PR DESCRIPTION
**Pull request by Ntu2koTrevor**

### CODE REVIEW

**Comment by Ntu2koTrevor** [comment]
@saeedmb8 Please review

File: gem/models.py [hunk 1]
```diff
@@ -68,6 +68,13 @@ class GemSettings(BaseSetting):
                   " line a break. Use only lowercase letters for keywords."
     )
 
+    moderator_name = models.TextField(
+        verbose_name='Moderator Name',
+        null=True,
+        blank=True,
+        help_text="Moderator name"
```
Comment (saeedmb8) [inline gem/models.py]: @Ntu2koTrevor can we change the help text to `This is the name that will appear on the front end when a moderator responds to a user`

**Comment by saeedmb8** [comment]
@Ntu2koTrevor I left a comment, you need to remove the migration and generate a new one once you have made that change

**Comment by Ntu2koTrevor** [comment]
gemsettings

**Comment by Ntu2koTrevor** [comment]
@saeedmb8

**Comment by saeedmb8** [comment]
👍  🍰 🍰 🍰 🍰

**Pull request merged**